.dockerignore

```diff
@@ -1 +1,2 @@
 .terraform
+.git
```

Comment:
is it intentional ?

Reply:
Yup! Git objects don't need to be in the docker context -- this line speeds up `docker build`s significantly